### PR TITLE
TASK: Hide DimensionSwitcher when empty

### PR DESCRIPTION
**Pull request by dfeyer**

### CODE REVIEW

File: packages/neos-ui/src/Containers/SecondaryToolbar/DimensionSwitcher/index.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ export default class DimensionSwitcher extends PureComponent {
         const contentDimensionsObject = contentDimensions.toObject();
         const contentDimensionsObjectKeys = Object.keys(contentDimensionsObject);
 
-        return (
+        return contentDimensionsObjectKeys.lenght ? (
```
Comment (dimaip) [inline packages/neos-ui/src/Containers/SecondaryToolbar/DimensionSwitcher/index.js]: `lenght` <-- a typo

File: packages/neos-ui/src/Containers/SecondaryToolbar/DimensionSwitcher/index.js [hunk 2]
```diff
@@ -117,7 +117,7 @@ export default class DimensionSwitcher extends PureComponent {
                     })}
                 </DropDown.Contents>
             </DropDown>
-        );
+        ) : <span />;
```
Comment (dimaip) [inline packages/neos-ui/src/Containers/SecondaryToolbar/DimensionSwitcher/index.js]: why do you need to return this span?

**Comment by dfeyer** [comment]
@dimaip Sorry for the typo and now i return null if empty

**Comment by dimaip** [comment]
Thanks!

**Pull request merged**